qam-minimal: enable s390tools test

tjyrinki opened the pull request:
qam-minimal: enable s390tools test

- Related ticket: https://progress.opensuse.org/issues/53012
- Verification runs:

15-SP2:
https://openqa.suse.de/t4546622

15-SP1:
https://openqa.suse.de/t4546572

15:
https://openqa.suse.de/t4546900

12-SP5:
https://openqa.suse.de/t4546623

12-SP4:
https://openqa.suse.de/t4546697

12-SP3:
https://openqa.suse.de/t4546698

12-SP2:
https://openqa.suse.de/t4546699

dzedro commented:
Don't put that into publish condition, it has nothing to do with it and it's confusing.

dzedro commented:
Why are you two doing on same issue ? 
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10737

tjyrinki commented:
Thanks for reviewing. We switched the task to me but it wasn't mentioned that there was existing PR.
I've modified to have an own section for it, is that better?
